ee/clickhouse/queries/person_distinct_id_query.py

```diff
@@ -1,12 +1,27 @@
-from django.conf import settings
+from datetime import timedelta
 
+from ee.clickhouse.materialized_columns.util import cache_for
 from ee.clickhouse.sql.person import GET_TEAM_PERSON_DISTINCT_IDS, GET_TEAM_PERSON_DISTINCT_IDS_NEW_TABLE
+from posthog.models.async_migration import is_async_migration_complete
+from posthog.settings import BENCHMARK, TEST
+
+using_new_table = TEST or BENCHMARK
 
 
 def get_team_distinct_ids_query(team_id: int) -> str:
     from ee.clickhouse.client import substitute_params
 
-    if str(team_id) in settings.PERSON_DISTINCT_ID_OPTIMIZATION_TEAM_IDS:
```

Comment:
Didn't we already enable it for a customer, if we did we shouldn't give them slow queries while we haven't ran the migration, we can just keep both conditions, either it's a listed ID or the migration is complete?

Might also make it easier to run the migration by team and rolling it out by team.

Reply:
We only merge this after migration has run on cloud so I believe this point is moot?